appveyor.yml

```diff
@@ -28,6 +32,25 @@ after_build:
 - cmd: '"C:\Program Files (x86)\CMake\bin\cpack.exe"'
 - ps: |
     Write-Output Get-FileHash SuperTux-*
+    
+    function transfer ($filename)
+    {
+        $file = Get-Item $filename;
+        invoke-webrequest -method put -infile $file.FullName https://transfer.sh/$($file.Name)
```

Comment:
AppVeyor does not support any sort of direct linking to the artifacts, which is unfortunate